identity/app/controllers/editprofile/ConsentsJourney.scala

```diff
@@ -60,8 +60,7 @@ trait ConsentsJourney
 
               case Right(updatedUser) =>
                 logger.info(s"Successfully set hasRepermissioned flag for user ${request.user.id}")
-                val page = IdentityPage("/complete-consents", "Complete Consents", isFlow = true)
-                consentCompleteView(page, returnUrl)
+                Future.successful(SeeOther(s"${routes.EditProfileController.displayConsentComplete().url}?returnUrl=${returnUrl}"))
```

Comment:
There's a`Redirect` method in play that takes a `queryString` argument which may be neater than this.